pywinauto/unittests/test_mouse.py

```diff
@@ -12,6 +12,14 @@
     from pywinauto.keyboard import SendKeys
     from pywinauto import mouse
     from pywinauto.timings import Timings
+elif sys.platform == 'darwin':
+    parent_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+    sys.path.append(parent_dir)
+    import mouse
```

Comment:
`from .. import mouse` not working? Using `sys.path.append` everywhere is not a good practice.